batch/test/advanced_creation.js

```diff
@@ -0,0 +1,276 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const path = require('path');
+const cp = require('child_process');
+const {describe, it, before} = require('mocha');
+const {BatchServiceClient} = require('@google-cloud/batch').v1;
+const batchClient = new BatchServiceClient();
```

Comment:
In other code I've seen, people frown a bit upon mixing the imports and the initializers, but this might be a YMMV thing, and it's test code, anyway.